tests/framework/extensions/machinepools/machinepools.go

```diff
@@ -4,12 +4,73 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"time"
 
 	apisV1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
 	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
 )
 
+const (
+	ProvisioningSteveResourceType = "provisioning.cattle.io.cluster"
```

Comment:
Don't we already have this defined somewhere. Why export it again?

Reply:
We have it in the `clusters` package, but if you reference it, then you introduce a circular dependency error with Go. At least that's what I saw when I tried it